web/src/components/tasks/LicenseStatusReceipt.tsx

```diff
@@ -77,34 +73,35 @@ const LicenseStatusLookup: Record<LicenseStatus, string> = {
   REVOKED: Config.licenseSearchTask.revokedStatusText,
 };
 
+const pendingStatuses: Set<LicenseStatus> = new Set([
+  "PENDING",
+  "DRAFT",
+  "SUBMITTED",
+  "UNDER_INTERNAL_REVIEW",
+  "SPECIAL_REVIEW",
+  "PENDING_DEFICIENCIES",
+  "DEFICIENCIES_SUBMITTED",
+  "CHECKLIST_COMPLETED",
+  "APPROVED",
+  "PENDING_RENEWAL",
+  "PENDING_REINSTATEMENT",
+]);
+
+const isPending = (licenseStatus: LicenseStatus): boolean => {
+  return pendingStatuses.has(licenseStatus);
+};
+
 export const LicenseStatusReceipt = (props: Props): ReactElement => {
-  const [theme, setTheme] = useState<PermitTheme>(pendingPermitTheme);
+  const [permitColorScheme, setPermitColorTheme] = useState<PermitTheme>(pendingPermitTheme);
```

Comment:
[sand] Should this be `permitColorTheme` rather than `permitColorScheme`, given that you use Theme in the setter function and in the default state?

Reply:
[pebble] Does this need to be a state variable? Is there a situation you envision that the variable data needs to be stateful and/or changed upon re-render?

Reply:
The change should have been from `[theme, setTheme]` to [permitColorScheme, setPermitColorScheme]` based on feedback that the (initial) variable names should "clarify what it's actually doing or what area of of the application it's actually impacting".